lib/jobFactory.js

```diff
@@ -46,8 +56,21 @@ class JobFactory extends BaseFactory {
 
         c.state = 'ENABLED';
         c.archived = false;
+        const annotations = c.permutations[0].annotations;
+
+        return super.create(c).then((job) => {
+            if (!annotations || !annotations['beta.screwdriver.cd/buildPeriodically']) {
```

Comment:
I can add it. The reason I didn't add it is because they are spread in 2 different files. This helper function will need a separate file, just for 1 line